src/libstd/process.rs

```diff
@@ -753,6 +754,34 @@ impl fmt::Debug for Stdio {
     }
 }
 
+#[stable(feature = "stdio_from", since = "1.19.0")]
```

Comment:
Note to self, these will need to become 1.20.0 if we don't make it before the upcoming branch.